conda-recipes/llvmdev_manylinux2014/meta.yaml

```diff
@@ -1,21 +1,8 @@
-# LLVM 10.0.0 and 10.0.1 had issues with aarch64
-{% if aarch64 %}
-
-{% set shortversion = "9.0" %}
-{% set version = "9.0.1" %}
-{% set sha256_llvm = "00a1ee1f389f81e9979f3a640a01c431b3021de0d42278f6508391a2f0b81c9a" %}
-{% set sha256_lld = "86262bad3e2fd784ba8c5e2158d7aa36f12b85f2515e95bc81d65d75bb9b0c82" %}
-{% set build_number = "0" %}
-
-{% else %}
-
-{% set shortversion = "10.0" %}
-{% set version = "10.0.1" %}
-{% set sha256_llvm = "c5d8e30b57cbded7128d78e5e8dad811bff97a8d471896812f57fa99ee82cdf3" %}
-{% set sha256_lld = "591449e0aa623a6318d5ce2371860401653c48bb540982ccdd933992cb88df7a" %}
-{% set build_number = "1" %}
-
-{% endif %}
+{% set shortversion = "11.1" %}
+{% set version = "11.1.0" %}
+{% set sha256_llvm = "ce8508e318a01a63d4e8b3090ab2ded3c598a50258cc49e2625b9120d4c03ea5" %}
+{% set sha256_lld = "017a788cbe1ecc4a949abf10755870519086d058a2e99f438829aef24f0c66ce" %}
+{% set build_number = "2" %}
```

Comment:
why build_number 2?

Reply:
This is historical, the other conda `llvmdev` (not the manylinux ones) also has build number two, so for coherence this build number was adapted here too.